src/fhir/1_Questionnaire-C-IDAS.json

```
@@ -1341,12 +1341,12 @@
       "type": "choice",
       "required": false,
       "linkId": "cs-idas-18",
-      "text": "Mood-congruent delusions - delusions of poverty, illness or loss",
+      "text": "In the last week have you observed <b>mood-congruent delusions</b> in your care recipient? (delusions of poverty, illness or loss)",
```

@sierramatice I missed this one, 
```suggestion
      "text": "In the last week have you observed mood-congruent delusions in your care recipient? (delusions of poverty, illness or loss)",
```